### ISSUE TITLE
useTemplateRef is not auto imported

### ISSUE THREAD

**Issue by amir20** [unplugin/unplugin-auto-import#526]
### Describe the bug

I am not sure what is different about `useTemplateRef` where it is not auto imported. 

I have the vue 3.5.2. 

```
AutoImport({
      imports: ["vue", VueRouterAutoImports, "vue-i18n", "pinia", "@vueuse/head", "@vueuse/core"],
      dts: "assets/auto-imports.d.ts",
      dirs: ["assets/composable", "assets/stores", "assets/utils"],
      vueTemplate: true,
    }),
```

`vue-tsc` produces: 

```
assets/components/ContainerViewer/ContainerLog.vue:48:16 - error TS2304: Cannot find name 'useTemplateRef'.

48 const viewer = useTemplateRef<ComponentExposed<typeof ViewerWithSource>>("viewer");
                  ~~~~~~~~~~~~~~


Found 1 error in assets/components/ContainerViewer/ContainerLog.vue:48
```

### Reproduction

I don't have one but I can try making one 

### System Info

```Shell
System:
    OS: macOS 14.6.1
    CPU: (12) arm64 Apple M3 Pro
    Memory: 127.41 MB / 18.00 GB
    Shell: 3.7.1 - /opt/homebrew/bin/fish
  Binaries:
    Node: 22.6.0 - ~/.local/state/fnm_multishells/92693_1725557701982/bin/node
    npm: 10.8.2 - ~/.local/state/fnm_multishells/92693_1725557701982/bin/npm
    pnpm: 9.9.0 - ~/.local/state/fnm_multishells/92693_1725557701982/bin/pnpm
    bun: 1.1.26 - /opt/homebrew/bin/bun
  Browsers:
    Chrome: 128.0.6613.115
    Safari: 17.6
```

### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by namefhf** [comment]
maybe it is not in https://github.com/unjs/unimport/blob/main/src/presets/vue.ts

**Comment by namefhf** [comment]
maybe it is not in https://github.com/unjs/unimport/blob/main/src/presets/vue.ts

**Comment by xlanghejianfeng** [comment]
今天突然build auto import 报错 error TS2304: Cannot find name 'ref'.

**Comment by lin-97** [comment]
大佬，我们都等着呢

**Comment by JianGuoHt** [comment]
help! ☕☕☕

**Comment by s3xysteak** [comment]
Also `useId()`

**Comment by li-junpeng** [comment]
先手动配置一下吧，还不知道什么时候能支持上呢

```ts
// vite.config.ts

import AutoImport from 'unplugin-auto-import/vite'

export default defineConfig({
  plugins: [
    AutoImport({
      imports: [
        'vue',
        {
          // vue 3.5.x
          vue: ['useTemplateRef', 'onWatcherCleanup', 'useId']
        }
      ]
    })
  ]
})
```

**Comment by s3xysteak** [comment]
Fixed in `v0.18.3`

**Comment by Lastcall9** [comment]
useTemplateRef return type is boolean?
![image](https://github.com/user-attachments/assets/8b14342a-fb75-482c-b812-6c166948e199)
